rust-src/concordium_base/src/smart_contracts.rs

```diff
@@ -4,17 +4,19 @@ use concordium_contracts_common::ModuleReference;
 /// Re-export of common helper functionality for smart contract, such as types
 /// and serialization specific for smart contracts.
 pub use concordium_contracts_common::{
-    self, ContractName, OwnedContractName, OwnedReceiveName, ReceiveName,
+    self, ContractName, ExceedsParameterSize, OwnedContractName, OwnedParameter, OwnedReceiveName,
+    ReceiveName,
 };
 use crypto_common::{
-    derive,
     derive::{Serial, Serialize},
     Buffer, Deserial, Get, ParseResult, ReadBytesExt, SerdeDeserialize, SerdeSerialize, Serial,
 };
 use derive_more::*;
 use sha2::Digest;
 use std::convert::{TryFrom, TryInto};
-use thiserror::Error;
+
+#[deprecated(note = "Replaced by `OwnedParameter` for consistency. Use that one instead.")]
```

Comment:
Can we make this link as well? Or is Markdown not supported here?

Reply:
But if we add a link (to docs.rs) (as `[OwnedParameter](https://docs.rs/....`) then it works right?

Reply:
Never mind. I was answering before you pushed :).

Looks good.